ccd-definition/ComplexTypes/RepaymentPlanLRspec.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "RepaymentPlanLRspec",
```

Comment:
We don't need to name these ID with suffix  LRspec to suppress them in live (apart from  Damages). The file name having LRspec suffix is enough. Another new file RepaymentFrequencyLRspec.json added in this PR does not have LRspec. It would be great if we can maintain consistency.

Reply:
I've amended RepaymentFrequencyLRspec.json, I want to keep json name as Id for my class.

Change incorporated